cpp/dolfinx/fem/DofMapBuilder.h

```diff
@@ -40,13 +40,15 @@ class DofMapBuilder
                     std::shared_ptr<const common::IndexMap>,
                     graph::AdjacencyList<std::int32_t>>
   build(MPI_Comm comm, const mesh::Topology& topology,
-        std::shared_ptr<const ElementDofLayout> element_dof_layout);
+        std::shared_ptr<const ElementDofLayout> element_dof_layout,
```

Comment:
Could you add a full doxygen descriptions for this constructor? I'm not sure from the interface alone what `transpose_blocks` is.